app/templates/main/index.html

```diff
@@ -1,17 +1,166 @@
 {% extends 'layouts/base.html' %}
 
 {% block content %}
-    <div class="ui text container">
-        <h1>Hello, world</h1>
-
-        <p>This is <a href="http://hack4impact.org">Hack4Impact</a>'s web application template. We'll use this as a
-        foundation for each of our <a href="http://flask.pocoo.org">Flask</a> projects. The goal is to reduce the work
-        necessary to get a new project off the ground, by providing boilerplate code (such as account management
-        facilities), as well as uncontroversial and extensible defaults. We have integrated an
-        <a href="http://sass-lang.com">SCSS</a>-based asset pipeline based around a (fantastic!) framework
-        called <a href="http://semantic-ui.com">Semantic UI</a>. Much of this code is appropriated from the examples
-        in Miguel Grinberg's book, <a href="http://flaskbook.com"><em>Flask Web Development</em></a>.</p>
-
-        <p>{{ lipsum(6) }}</p>
+<html>
+  <h1> Maps4All</h1>
+  <head>
+    <title>Place Autocomplete</title>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+    <meta charset="utf-8">
+    <style>
```

Comment:
Please place CSS and Javascript in separate files (here and everywhere else). For this project, the directory for CSS files is app/assets/styles and the directory for Javascript files is app/assets/scripts. Since both of these seem to be provided by a 3rd-party library (correct me if I'm wrong), you would also want to place the script inside the 'vendor' directories within 'styles' and 'scripts' and name it after the 3rd-party library's name (e.g. if the files are from gmaps, then gmaps.scss and gmaps.js).

Please also note that we're using SASS (http://sass-lang.com/), which is a CSS-like language - that's why the extension is scss and not just css. It is used to avoid writing repetitive CSS and it itself compiles to CSS that your app will eventually use. SASS and CSS syntax is very similar - I think the only thing you should be look out for is nesting (e.g. see "Nesting" here: http://sass-lang.com/guide)